### PR TITLE
/#1653: Implement accented characters support

### PR DESCRIPTION
**Pull request by oderayi**

### CODE REVIEW

File: src/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mojaloop/sdk-scheme-adapter",
-  "version": "11.2.4",
+  "version": "11.2.5",
```
Comment (vgenev) [inline src/package.json:3]: I think in the new versioning model we need to change the version to 11.3.0, @elnyry-sam-k , @rmothilal can you confirm?

**Review by vgenev — CHANGES_REQUESTED** [review]

File: src/package.json (hunk 1 above)
Comment (elnyry-sam-k) [reply src/package.json:3]: yes that is correct. this is new non-breaking functionality, so 11.3.0

**Review by rmothilal — APPROVED** [review]
HAppy

**Review by bushjames — COMMENTED** [review]
Nice work @oderayi !

**Pull request merged**